egs/aspire/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -21,14 +21,29 @@ set -e
 local/multi_condition/check_version.sh || exit 1;
 
 mkdir -p exp/nnet3
-if [ $stage -le 1 ]; then
+if [ $stage -le 0 ]; then
   # prepare the impulse responses
   local/multi_condition/prepare_impulses_noises.sh --log-dir exp/make_reverb/log \
     --db-string "$db_string" \
     --download-rirs $download_rirs \
     --RIR-home $RIR_home \
     data/impulses_noises || exit 1;
 
+  # Generate the rir_list and noise_list for the reverberate_data_dir.py to corrupt the data
+  # this script just assumes air rwcp rvb2014 databases
+  python local/multi_condition/aspire_prep_rir_noise_list.py data/impulses_noises data/impulses_noises/info
```

Comment:
given that we are downloading several databases now, you could probably move all these to a new directory like `s5/databases`, rather than storing everything in the `data/` directory.